deploy/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: observatorium-operator
```

Comment:
could we as a follow up generate these with jsonnet as well?

Reply:
Done

Reply:
Should this be done in this PR or a follow-up one?

Reply:
happy to have this be a follow up